PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -442,6 +452,25 @@ struct FlowGFWPbPb {
     registry.fill(HIST("BeforeCut_multT0C_centT0C"), collision.centFT0C(), collision.multFT0C());
     registry.fill(HIST("hEventCount"), 1.5);
 
+    Int_t multITSnoTPC = 0, multITSonly = 0, multITSTPC = 0;
+
+    for (auto& track : tracks) {
+
+      if (track.hasITS() && !track.hasTPC())
+        multITSnoTPC++;
+
+      if (track.hasITS())
+        multITSonly++;
+
+      if (track.hasITS() && track.hasTPC())
+        multITSTPC++;
+    }
```

Comment:
Has this code run locally and checked that it is getting the expected results?
Usually, if global tracks are required, they all have ITS and TPC by construction

Reply:
Thank you for your comment! My intention here is to see how the global tracks differ from ITS only by plotting histograms. Then, I would remove the requirement of global tracks and retest. My idea is to try to figure out why the elliptic flow from Pb-Pb Run 3 ends up below Run 2. The attached plot might help.
<img width="375" alt="image" src="https://github.com/AliceO2Group/O2Physics/assets/152337132/1ff1b938-9a92-4e32-b2ec-480117ac7478">